### PR TITLE
Undefined value symbol as a parameter (instead of an em dash constant)

### PR DESCRIPTION
**Pull request by So-Fras**
Signed-off-by: Sophie Frasnedo <sophie.frasnedo@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#471

**Other information**:
In SLD, the valueFormatter (where the "undefined value symbol" is set as a parameter) is created in the LayoutParameters class.  In a future release, a new SvgParameter class will be created (see #469) for Single Line Diagram. The valueFormatter creation shall then be moved in the newly created class

### CODE REVIEW

File: network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java [hunk 1]
```diff
@@ -54,6 +54,9 @@ public class SvgParameters {
     private int angleValuePrecision = 1;
     private double pstArrowHeadSize = 8;
 
+    /** em dash unicode for undefined value */
+    private String undefinedValueSymbol = "\u2014";
```
Comment (flo-dup) [inline network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java:58]: Could you replace this by `""`? it doesn't really look good a dash above an edge, and now we can take advantage of this new feature!

**Review by flo-dup — COMMENTED** [review]
Great! juste one minor comment

File: network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java (hunk 1 above)
Comment (So-Fras) [reply network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java]: Yes, my bad, I forgot we talked about that !

**Review by flo-dup — CHANGES_REQUESTED** [review]

File: network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java [hunk 2]
```diff
@@ -54,6 +54,9 @@ public class SvgParameters {
     private int angleValuePrecision = 1;
     private double pstArrowHeadSize = 8;
 
+    /** em dash unicode for undefined value */
```
[the remark this answers is not in the document]
Comment (So-Fras) [reply network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgParameters.java:57]: yes indeed...

**Review by flo-dup — APPROVED** [review]

**Pull request merged**